src/trading-platform-pp.pro

```diff
@@ -11,7 +11,8 @@ CONFIG += c++11
 SOURCES += \
     api/api.cpp \
     main.cpp \
-    gui/mainwindow.cpp
+    gui/mainwindow.cpp \
+    style.astylerc
```

Comment:
I don't think this is necessary.

Reply:
This is necessary to activate the formatter in Qt Creator

Reply:
We should use DISTFILES variable instead. Take a look at https://doc.qt.io/qtcreator/creator-beautifier.html section 11, "Select the Use file defined in project files"

Reply:
Have you tried to put it in DISTFILES? It only works for me when I put it in SOURCES.

Reply:
It works well for me. Everytime I edit and save a file, Qt Creator automatically formats the file.